lib/FCM/System/Make/Share/Config.pm

```diff
@@ -217,6 +219,18 @@ sub _parse_name {
     $m_ctx->set_name($entry->get_value());
 }
 
+# Parse "require-version" declaration from a config entry.
+sub _parse_require_version {
+    my ($attrib_ref, $m_ctx, $entry) = @_;
+    my ($version) = shellwords($attrib_ref->{util}->version());
+    my ($min_version, $max_version) = shellwords($entry->get_value());
+    if (    $min_version gt $version
+        ||  defined($max_version) && $version gt $max_version
+    ) {
```

Comment:
That sounds about right. (Obviously, the numeric comparison will not work if we issue more than 10 formal releases in a single month, but I don't think we'll ever get to that for FCM.)